src/ex2/ItemManager.js

```diff
@@ -0,0 +1,43 @@
+export default class ItemManager {
+  constructor() {
+    {
+      this.TasksArray = this.getTasksFromLocalStorage();
+    }
+  }
+ 
+  addItem(newTasks) {
+    this.TasksArray.push(newTasks);
+    localStorage.setItem(
+      "tasks",
+      JSON.stringify([
+        ...JSON.parse(localStorage.getItem("tasks") || "[]"),
+        newTasks,
+      ])
+    );
+  }
+
+  tasksArray() {
+    return this.TasksArray;
+  }
+  taskToRmove(taskToRemove) {
```

Comment:
better call method `removeTask`

Reply:
method param should be named with not ambiguously (in this case `taskIndexToRemove` may be better)